data-model/annotations/schema/annotation.json

```diff
@@ -161,27 +246,48 @@
     },
     "dcterms:created": {
       "type": "string",
-      "format": "date-time"
+      "format": "date-time",
+      "description": "The date and time when the annotation was created. http://purl.org/dc/terms/created"
     },
     "ods:deletedOn": {
       "type": "string",
-      "format": "date-time"
+      "format": "date-time",
+      "description": "Filled if the annotation was archived. The date and time when the annotation was archived"
     },
     "oa:generated": {
       "type": "string",
-      "format": "date-time"
+      "format": "date-time",
+      "description": "The date and time when the annotation was generated. http://purl.org/dc/terms/issued",
+      "$comment": "Be aware this is not a valid oa field. This is called generated within W3C annotation model but actually maps to dcterms:issued. We kept it generated to keep in line with W3C annotation standard."
```

Comment:
If it's not a valid oa:field, should it be in the oa namespace? should we move it to ods?

Reply:
We discussed this, but we don't have a solution. We could use the dc:terms to which it refers, but then we might lose the W3C compliancy. I don't think we should create our own term. That is why I put in the comment here, it needs another looks to check how to do this right.